vividus-analytics/src/main/resources/spring.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:util="http://www.springframework.org/schema/util"
+    xsi:schemaLocation="
+        http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/util https://www.springframework.org/schema/util/spring-util.xsd"
+    default-lazy-init="true">
+
+    <bean id="analyticsFacade" init-method="init" class="org.vividus.analytics.GoogleAnalyticsFacade">
+        <property name="trackingId" value="${analytics.tracking-id}" />
+        <property name="analyticsUri" value="${analytics.uri}" />
+    </bean>
+
+    <bean id="analyticsService" class="org.vividus.analytics.AnalyticsService" lazy-init="false">
```

Comment:
`analyticsService` depends on `analyticsFacade`, so service bean will be created at the context start (not lazily), the creation of service bean will trigger creation of facade bean
having all this said:
- `default-lazy-init="true"` can be removed from root
- `lazy-init="false"` can be removed from this service bean